lib/DriverTool/autolink_extract_main.cpp

```diff
@@ -229,6 +252,13 @@ int autolink_extract_main(ArrayRef<const char *> Args, const char *Argv0,
 
   std::vector<std::string> LinkerFlags;
 
+  // Keep track of whether we've already added the common
+  // Swift libraries that ususally have autolink directives
+  // in most object fiels
+  std::unordered_map<std::string, bool> CommonAddedSwiftLibraries = {{"-lswiftSwiftOnoneSupport", false},
+                                                                    {"-lswiftCore", false},
+                                                                    {"-lswift_Concurrency", false}};
```

Comment:
Adding a `,` after this last entry and clang-formatting would make this more readable IMO.